packages/gasket-cli/bin/boot.js

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env node
-
 require('dotenv').config();
 const { join } = require('path');
-const location = ['@gasket', 'cli', 'bin', 'run'];
+const location = ['@gasket', 'cli', 'lib', 'bin', 'run'];
```

Comment:
```suggestion
const location = ['@gasket', 'cli', 'bin', 'run'];
```

Reply:
I doubled checked this locally with the extension change. It appears to work ok as long as we `require` and run has a `.js` extension. We may need to reevaluate this should we change boot/run to ESM.